changelogs/unreleased/5218-support-archive.yml

```diff
@@ -0,0 +1,6 @@
+change-type: patch
+description: "Update support archive link for v2"
+issue-nr: 5218
+destination-branches:
```

Comment:
Can you add a section - minor-improvement for this one? It won't be displayed in the outputed changelog otherwise.